glossary.html

```diff
@@ -4,30 +4,38 @@
     <title>sanjay’s CSD120</title>
 </head>
 <body>
+         
+    <header>
         <nav>
-        <ul>
-        <li><a href="index.html">home</a> </li>
-        <li><a href="links.html">Links</a> </li>
-        <li><a href="glossary.html">Grossary</a></li>
-        </ul>
+            <ul>
+                <li><a href="index.html">Home</a> </li>
+                <li><a href="links.html">Links</a> </li>
+                <li><a href="glossary.html">Glossary</a></li>
+            </ul>
         </nav>
+    </header>
+
+    <main>
         <h1>Acronym Glossary</h1>
-        <dt>
-        <dt>HTML</dt>
-        <dd>Hyper Text Markup Language</dd>
-        <dt>HTTP</dt>
-        <dd>Hypertext Transfer Protocol</dd>
-        <dt>IP</dt>
-        <dd>Internet Protocol</dd>
-        <dt>TCP</dt>
-        <dd>Transmission Control Protocol</dd>
-        <dt>URL</dt>
-        <dd>Uniform Resource Locator</dd>
-        </dt>
-        </main>
+        <dl>
```

Comment:
The `<dl>` is the main definition list element that should wrap all `<dt>` and `<dd>` element pairs